web/cypress/fixtures/export-route-dataset.json

```diff
@@ -131,7 +129,6 @@
             }
           },
           "x-apisix-enable_websocket": false,
-          "x-apisix-plugins": {},
```

Comment:
Just want to know why delete it?

Reply:
The design of import and export is to export when there is data, and not export when there is no data. Here, no data is empty. I don't think it's necessary to export, so I delete it.

Reply:
If it is not deleted, Fe ci will report an error.

Reply:
@Jaycean do we have some test case for `x-apisix-plugins` not empty ?

Reply:
Yes, most of the other test cases have data in `x-apisix-plugins`
For example, this test case: 
https://github.com/apache/apisix-dashboard/blob/73e96357c64ddc92551854577879b86a4ecee341/api/internal/handler/data_loader/route_export_test.go#L34

Reply:
Thank you for the explanation.Got it.